internal/worker/worker.go

```diff
@@ -116,6 +116,9 @@ func (worker *Worker) registerWorker(ctx context.Context) error {
 		Meta: v1.Meta{
 			Name: worker.name,
 		},
+		Resources: map[string]uint64{
+			v1.ResourceTartVMs: 2,
```

Comment:
Let's also make a VM specify this default resource as 1. So scheduling will be working appropriately by default.

Reply:
> Let's also make a VM specify this default resource as 1. So scheduling will be working appropriately by default.

Implemented in https://github.com/cirruslabs/orchard/pull/63/commits/456c713311b88230eede04b87bf0ff7e5bc140bb.